.travis.yml

```diff
@@ -107,6 +107,7 @@ install:
   - pip install coveralls
 
 script:
+  - python -m pyproj  # test __main__ module
```

Comment:
Is this needed? (I think the somewhat longer alternative is `python -c "import pyproj; pyproj.show_versions()"`?) 

Just asking, as the alternative is not really cumbersome to me (and actually more explicit that you are printing the versions), and adding it is yet another corner feature that needs to be supported (as external users might start to use it as well)